RockLib.DistributedTracing.AspNetCore/AspNetCore/HttpContextExtensions.cs

```diff
@@ -46,16 +50,25 @@ public static ICorrelationIdAccessor GetCorrelationIdAccessor(this HttpContext h
             throw new ArgumentNullException(nameof(correlationIdHeader));
          }
 
-         if (!httpContext.Items.TryGetValue(typeof(ICorrelationIdAccessor), out var value)
-             || value is not ICorrelationIdAccessor accessor)
+         if (!httpContext.Items.TryGetValue(typeof(ICorrelationIdAccessor), out var value) 
+            || value is not ICorrelationIdAccessor accessor)
          {
             accessor = new CorrelationIdAccessor();
 
+            // align w/OpenTelemetry
+            accessor.TraceId = GetTraceId(httpContext);
```

Comment:
I think it would make sense to leave CorrelationId separate from trace and span.  What do you think?

Reply:
Nm.  Thats in the GetAccessor.